frontend/src/components/common/player/common/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const MILLISECONDS_IN_SECOND = 1000;
+export const MAX_RATE_STEP_ANGLE_IN_DEGREES = 45;
+export const MIDDLE_RATE_STEP = 1;
+export const RATE_STEPS = ['0.25', '0.5', '0.75', '1.0', '1.25', '1.5', '1.75'];
```

Comment:
why are they strings?

Reply:
> why are they strings?

Because I want to show decimal values in UI. If I use array of numbers, some values (for example 1) will be integer